packages/web-react/src/components/Select/stories/argTypes.ts

```diff
@@ -0,0 +1,61 @@
+import { ValidationStates } from '../../../constants';
+
+export default {
+  validationState: {
+    control: {
+      type: 'select',
+      options: [...Object.values(ValidationStates), undefined],
+    },
+    defaultValue: undefined,
+  },
+  isDisabled: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  isRequired: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  isLabelHidden: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  isFluid: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  label: {
+    control: {
+      type: 'text',
+    },
+    defaultValue: 'Field label',
+  },
+  name: {
+    control: {
+      type: 'text',
+    },
+    defaultValue: 'example',
+  },
+  validationText: {
+    control: {
+      type: 'object',
+    },
+    defaultValue: '',
+    description:
+      'The validation message. Use a string `"foo"` for single message or an array for multiple messages `["foo", "bar"]`.',
```

Comment:
FYI I made a mistake and this should be 
```suggestion
      'The validation text. Use a string `"foo"` for single message or an array for multiple messages `["foo", "bar"]`.',
```

You set it correctly and we should fix this in the future. Now it is confusing (my bad), but it is consistent with other components (yours good job).

Reply:
Are you kidding me?

Reply:
So now it should be like this? "The validation text. Use a string `"foo"` for single text or an array for multiple texts `["foo", "bar"]`."

Reply:
Don't change it now. We will change this in the future. It is my mistake, you did it correctly here.